ios/Podfile.lock

```diff
@@ -371,7 +371,7 @@ PODS:
     - React-Core
   - react-native-keep-awake (4.0.0):
     - React
-  - react-native-netinfo (9.4.1):
```

Comment:
Didn't this one require the API changes?

Reply:
I don't recall that.